test/rxd/conftest.py

```diff
@@ -44,6 +44,8 @@ def neuron_nosave_instance(neuron_import):
     # out of scope after neuron_instance is torn down.
     # Here we assert that no section left alive. If the assertion fails it is
     # due to a problem with the previous test, not the test which failed.
+    for sec in h.allsec():
+        h.delete_section(sec=sec)
```

Comment:
@adamjhn I'm of two minds about doing this. In favor, it allows all the tests to run instead of stopping at the first error.  It is nice to see all the tolerance errors. But what's your opinion?